src/components/text-field/react/TextField.tsx

```diff
@@ -77,6 +83,12 @@ const COMPONENT_NAME = `${COMPONENT_PREFIX}TextField`;
  */
 const CLASSNAME: string = getRootClassName(COMPONENT_NAME);
 
+/**
+ * The line height for the Text Area. Please keep in mind that if that changes via css, the prop lineHeight needs to be changed
```

Comment:
There is a library that can calculate the line height of any element dynamically: https://www.npmjs.com/package/line-height
@juanigalan91 Do you think it could be a good option here?

Reply:
It would be nice to test. Trying to modify the line height in css and check that this lib can calculate the correct line height with reasonable performance

Reply:
I was just looking for a way to avoid the hardcoding and apparently we can use `event.target.clientHeight`. I already tested it and seems to be working as expected. and maybe we can avoid the library :)

Reply:
Ok nice job :+1: